backend/kernelCI_app/management/commands/helpers/kcidbng_ingester.py

```diff
@@ -35,11 +38,40 @@
 
 logger = logging.getLogger("ingester")
 
-# Thread-safe queue for database operations
-db_queue = Queue()
+# Batching and backpressure controls
+try:
+    INGEST_BATCH_SIZE = int(os.environ.get("INGEST_BATCH_SIZE", "1000"))
+except Exception:
```

Comment:
nit: prefer being more specific (applies to lines below)

```suggestion
except (ValueError, TypeError):
```